mon-pix/app/components/reached-stage.js

```diff
@@ -0,0 +1,16 @@
+import Component from '@glimmer/component';
+import range from 'lodash/range';
+
+export default class ReachedStage extends Component {
+
+  get acquiredStars() {
+    // returns an array of [1...N] where N is the number of acquired stars
+    return range(1, this.args.starCount + 1);
```

Comment:
```suggestion
    return range(1, this.args. reachedStage + 1);
```
Dans le cas des étoiles pour les paliers acquis, on regarde le palier atteint, non ?

Reply:
`this.args.starCount` est une string. Du coup, "3" + 1 = 31